docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/digitalocean-app-platform-2022.md

```diff
@@ -207,19 +205,125 @@ At the top of the *Settings* menu select your Strapi application component. Scro
+### Deploy and access a Strapi application
 
 When the above steps are completed DigitalOcean should automatically try to build and deploy the application. If the build does not start, click the **Actions** button in the upper right and select *force rebuild and deploy*. The Strapi admin panel is accessed at {your app domain}/admin once the application is successfully deployed.
 
+## Add a managed database
+
+DigitalOcean managed databases are a production-scale database solution for a deployed Strapi application. Switching from a development database to a managed database requires modifying the Strapi application and modifying the settings on DigitalOcean.
+
+- removing the database environment variables,
+- installing the `pg-connection-string` dependency,
+- changing the production `database` file,
+- creating and attaching the database in the DigitalOcean platform.
+
+### Modify the Strapi application
+
+When a managed database is attached to a Strapi application, the connection parameters are passed directly from the database to the application `.yaml` file. This requires a modification to the `config/env/production/database` file and the addition of the `pg-connection-string` dependency.
+
+To add the `pg-connection-string` dependency navigate to the project directory and run the code snippet below in a terminal.
+
+<code-group>
+
+<code-block title="NPM">
+```bash
+npm install pg-connection-string --save <!-- needed?-->
+```
+</code-block>
+
+<code-block title="YARN">
+```bash
+yarn add pg-connection-string
+```
+</code-block>
+
+</code-group>
+
+To switch to a managed database change the `config/env/production/database` file to be able to parse the `DATABASE_URL` like the example below.
+
+<code-group>
+
+<code-block title='JAVASCRIPT'>
+
+```jsx
+// path: ./config/env/production/database.js
+
+const parse = require("pg-connection-string").parse;
+
+const { host, port, database, user, password } = parse(
+  process.env.DATABASE_URL
+);
+
+module.exports = ({ env }) => ({
+    connection: {
+      client: 'postgres',
+      cconnection: {
+        host,
+        port,
+        database,
+        user,
+        password,
+        ssl: {
+          rejectUnauthorized: false,
+        },
+      },
+      debug: false,
+    },
+  });
+
+```
+
+</code-block>
+
+<code-block title='TYPESCRIPT'>
+
+```jsx
+//path:config/env/production/database.ts
+
+const parse = require("pg-connection-string").parse;
+
+const { host, port, database, user, password } = parse(
+  process.env.DATABASE_URL
+);
+
+export default ({ env }) => ({
+    connection: {
+      client: 'postgres',
+      cconnection: {
+        host,
+        port,
+        database,
+        user,
+        password,
+        ssl: {
+          rejectUnauthorized: false,
+        },
+      },
+      debug: false,
+    },
+  });
+
+```
+
+</code-block>
+</code-group>
+
+### Modify settings on DigitalOcean
+
+1. Create a managed database.
+2. Attached the managed database to the app.
```

Comment:
```suggestion
2. Attach the managed database to the application.
```

Reply:
Sorry, this section was WIP and I should have marked it as such. fixed the typo. I will stick with "App" for the DigitalOcean product, since that is what they use.